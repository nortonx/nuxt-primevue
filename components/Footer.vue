<template>
  <footer>
    <p>&copy; {{ new Date().getFullYear() }}</p>
  </footer> 
</template>