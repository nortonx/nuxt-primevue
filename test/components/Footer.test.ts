import { mount } from 'vitest';
import Footer from '@/components/Footer.vue';

test('Footer component renders correctly', () => {
  const wrapper = mount(Footer);
  expect(wrapper.html()).toMatchSnapshot();
});
