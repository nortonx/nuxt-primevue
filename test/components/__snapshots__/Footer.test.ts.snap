// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > should render correctly and match snapshot 1`] = `
"<footer class="text-700 text-center">
  <p class="footer-text">Â© 2024</p>
</footer>"
`;
